### PR TITLE
Please add my lesson 7 homework

### PR DESCRIPTION
**Pull request by zoldirazvan**

### CODE REVIEW

File: lesson_7/lesson_7_zoldi_razvan/lesson_7_zoldi_razvan.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+# all the decorated functions names.
+import inspect
+
+print_registry = []
+
+
+# original code at: https://stackoverflow.com/questions/52191968/check-if-a-function-was-called-as-a-decorator
+# documentation at https://docs.python.org/3/library/inspect.html
+def register1(fnc):
+    lines = inspect.stack(context=2)[1].code_context
```
Comment (mariuspodean) [inline lesson_7/lesson_7_zoldi_razvan/lesson_7_zoldi_razvan.py]: Cool! It's great that you're using additional tools to help you with the assignment

File: lesson_7/turcu_alexandru/decorators.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+def uppercase(greetings):
```
Comment (mariuspodean) [inline lesson_7/turcu_alexandru/decorators.py]: who's file is this? Is it yours?

**Review by mariuspodean — CHANGES_REQUESTED** [review]
The folder structure should be:
lesson_7/name_surname/

File: lesson_7/turcu_alexandru/decorators.py (hunk 2 above)
Comment (zoldirazvan) [reply lesson_7/turcu_alexandru/decorators.py]: yes. when we receive multiple problems I always try to solve them step-by-step. after solving one I comment it and start another. I proposed myself to un-comment them at the end but I forgot.
lesson_7_zoldi_razvan

Comment (mariuspodean) [reply lesson_7/turcu_alexandru/decorators.py]: I was referring to the fact that the file has the following name: "lesson_7/turcu_alexandru/decorators.py "

Comment (zoldirazvan) [reply lesson_7/turcu_alexandru/decorators.py]: To be honest I'm not sure how I did that. Yes, your're right, that code is not mine.

**Review by mariuspodean — APPROVED** [review]

**Pull request merged**